test/gtest/common/test_obj_size.cc

```diff
@@ -47,7 +47,7 @@ UCS_TEST_F(test_obj_size, size) {
     UCS_TEST_SKIP_R("Assert enabled");
 #else
     EXPECTED_SIZE(ucp_ep_t, 64);
-    EXPECTED_SIZE(ucp_request_t, 232);
+    EXPECTED_SIZE(ucp_request_t, 240);
```

Comment:
can we avoid this?

Reply:
nop :(
we have to store user_data pointer